src/github.com/matrix-org/dendrite/cmd/dendrite-clientapi/main.go

```diff
@@ -52,9 +69,9 @@ func main() {
 		ServerName:           "localhost",
 		KeyID:                "ed25519:something",
 		PrivateKey:           privKey,
-		KafkaProducerURIs:    []string{"localhost:9092"},
-		ClientAPIOutputTopic: "roomserverInput",
-		RoomserverURL:        "http://localhost:7777",
+		KafkaProducerURIs:    kafkaURIs,
+		ClientAPIOutputTopic: clientAPIOutputTopic,
```

Comment:
Yeah, will need to have a think about this as it's only going to get worse. I feel it needs to be agnostic to the thing which is producing/consuming them (i.e remove the input/output distinction) but I don't know what a clear name would be for some topics..